tests/src/test/scala/org/apache/openwhisk/core/limits/ActionLimitsTests.scala

```diff
@@ -463,4 +463,34 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers with WskActorSys
       _.response.result.get.fields("error") shouldBe Messages.memoryExhausted.toJson
     }
   }
+
+  /**
+   * Test that a heavy logging action is interrupted within its timeout limits.
+   */
+  it should "interrupt the heavy logging action within its time limits" in withAssetCleaner(wskprops) {
+    (wp, assetHelper) =>
+      val name = "NodeJsTestLoggingActionCausingTimeout-" + System.currentTimeMillis()
```

Comment:
Alternatively can do string concatenation like the next line. Ex:
`val name = s"NodeJsTestLoggingActionCausingTimeout-${System.currentTimeMillis()}"`

Reply:
done, applied your proposal to the code